crates/pallets/starknet/src/tests/invoke_tx.rs

```diff
@@ -270,14 +270,16 @@ fn given_hardcoded_contract_run_storage_read_and_write_it_works() {
         basic_test_setup(2);
 
         let none_origin = RuntimeOrigin::none();
+        let contract_content = include_bytes!("../../../../../cairo-contracts/build/NoValidateAccount.json");
+
+        let mut transaction = get_storage_read_write_dummy();
+        let raw_contract_class: ContractClass =
+            ContractClass::V0(serde_json::from_slice(contract_content).expect("invalid contract content"));
+
+        transaction.contract_class =
+            Some(ContractClassWrapper::try_from(raw_contract_class).expect("invalid contract content"));
```

Comment:
should i add this logic be handled by `get_storage_read_write_dummy` function?

this was taken from `transaction_from_json` function

Reply:
you could just pass it as an option, would be better for DRY

Reply:
@EvolveArt i just moved it inside the function because we are only using that in 1 place and don't think we will require that transaction without contract class